src/Symfony/Component/String/AbstractUnicodeString.php

```diff
@@ -149,6 +149,13 @@ public function ascii(array $rules = []): self
         return $str;
     }
 
+    public function bytesAt(int $offset): array
```

Comment:
this implementation could be moved to `AbstractString` since it's generic (and we'd keep the optimized version in `ByteString`)